extras/create_send_notifications.py

```diff
@@ -107,12 +121,12 @@ def main():
     os.chdir(os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR))
 
     # Build list of CyHy orgs
```

Comment:
Update comment wording to match the changes in this PR:
```suggestion
    # Build list of orgs that should receive notifications
```